bonk/src/devices/powermate/events.ts

```diff
@@ -1,11 +1,26 @@
 /** Events for PowerMate button and rotation */
-export const PowerMateEvents: { [eventName: string]: string } = {
-  SINGLE_PRESS: 'singlePress',
-  DOUBLE_PRESS: 'doublePress',
-  TRIPLE_PRESS: 'triplePress',
-  LONG_PRESS: 'longPress',
-  CLOCKWISE: 'clockwise',
-  COUNTERCLOCKWISE: 'counterclockwise',
-  PRESS_CLOCKWISE: 'pressClockwise',
-  PRESS_COUNTERCLOCKWISE: 'pressCounterclockwise',
-};
+export const PowerMateEvents = [
+  'singlePress',
+  'doublePress',
+  'triplePress',
+  'longPress',
+  'clockwise',
+  'counterclockwise',
+  'pressClockwise',
+  'pressCounterclockwise',
+];
```

Comment:
I wish this didn't have to exist

This is only because I can't iterate through the keys of an interface in TypeScript

I went Googling and found the `ts-transformer-keys` thing, and I was able to get it working in the editor, but running the code with node failed because Rollup didn't compile it right (their docs say you have to modify the build/compilation/etc in your project to actually make it work) https://github.com/kimamula/ts-transformer-keys